01-import_and_filter.py

```diff
@@ -78,18 +78,17 @@ def run_filter(subject):
             raw.resample(config.resample_sfreq, npad='auto')
 
         raw.save(raw_fname_out, overwrite=True)
-        raws.append(raw)
```

Comment:
@SophieHerbst this avoids keeping all the raw files in memory just to know how many you have seen

Reply:
great!